Modules/vc-module-export/VirtoCommerce.ExportModule.Web/Scripts/blades/export-type-selector.js

```diff
@@ -35,6 +35,7 @@ angular.module('virtoCommerce.exportModule')
                     var exportDataRequest = {
                         exportTypeName: selectedType.typeName,
                         allColumnsOfType: allColumnsOfType,
+                        isTabularExportSupported: selectedType.isTabularExportSupported,
```

Comment:
Let's move `isTabularExportSupported` property to independent parameter to callback, to not pass it to server with exportDataRequest.
Same will be done for `allColumnsOfType` in another PR.

Reply:
I'm already done it for allColumnsOfType

Reply:
@akak1977 Please take this changes into your branch, as it is already merged.